CONTRIBUTING.md

```diff
@@ -6,6 +6,10 @@ Before submitting new features or changes to current functionality, it is recomm
 [open an issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/new)
 and discuss your ideas or propose the changes you wish to make.
 
+**Breaking Changes**: If your PR introduces a breaking change, please add the `breaking change` label.
```

Comment:
I think this could be moved here:

- **When to Use**:
  - API changes that break backward compatibility
  - Configuration changes that require user action
  - Behavioral changes that might affect existing users
  - Removal of deprecated features

Reply:
and add Deprecations here too